build.sbt

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
1. Don't forget to update `version` to 0.2.0 (in first commit in branch). 
2. Create temporary commit "Bump to rc1" as last in branch, after "Prepare for release"
3. Add git tag for this commit  "0.2.0-rc1"
4. Push, Travis will deploy it to bintray.

**Note**: do not create "0.2.0" tag yet.